src/components/institution/OrganizationTree.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchOrganization } from '../../api/cristinApi';
+import { getOrganizationHierarchy } from '../../utils/institutions-helpers';
+import { getLanguageString } from '../../utils/translation-helpers';
+import { AffiliationSkeleton } from './AffiliationSkeleton';
+
+interface AffiliationHierarchyProps {
+  unitUri: string;
+}
+
+export const OrganizationTree = ({ unitUri }: AffiliationHierarchyProps) => {
+  const { t } = useTranslation();
+
+  const organizationQuery = useQuery({
+    queryKey: [unitUri],
+    queryFn: () => fetchOrganization(unitUri),
+    meta: { errorMessage: t('feedback.error.get_institution') },
+    staleTime: Infinity,
+    cacheTime: 1_800_000, // 30 minutes
+  });
+  const organization = organizationQuery.data;
+
+  const unitNames = getOrganizationHierarchy(organizationQuery.data).map((unit) => getLanguageString(unit.labels));
```

Comment:
Gir det mening å beholde hele enheten, og så bruke `id` som `key` i stedet for unitName? Samme med at vi da kan bruke country fra objektet som er i mappen, i stedet for at den kommer utenfra på en litt "mystisk" måte
```suggestion
  const units = getOrganizationHierarchy(organizationQuery.data);
```

Reply:
er enig i at det gir mening, og prøvde det først. Men av en eller annen grunn får kun det siste elementet i arrayet som returneres av `getOrganizationHierarchy` `country` 🤔  Har gjort om til å mappe `units` uansett, da det er mer naturlig å bruke `id` som key imo

Reply:
hmm, virker kanskje som en (egentlig urelatert) bug 🤔 country skal bære være på toppnivå for en org, men her kommer den på bunnivå 😅 
Kan være at `getOrganizationHierarchy` gjør noe dumt

Reply:
Ser nå at man får country for den enheten man slår opp, ikke bare på toppnivå som jeg trodde. Da gir det mening!

Reply:
Da gjør du det du selv syns gir mest mening her :)